plugin/evm/orderbook_test.go

```diff
@@ -796,7 +796,7 @@ var (
 		  }
 		]
 	  }`
-	orderBookAddress common.Address = common.HexToAddress("0x0300000000000000000000000000000000000069")
+	orderBookAddress common.Address = common.HexToAddress("0x0300000000000000000000000000000000000000")
```

Comment:
also need to change this at line 807 in the genesis json string

Reply:
done